cpp/include/cudf/strings/strings_column_view.hpp

```diff
@@ -106,20 +106,23 @@ class strings_column_view : private column_view {
   /**
    * @brief Returns the internal column of chars
    *
-   * @throw cudf::logic_error if this is an empty column
+   * @throw cudf::logic error if this is an empty column
+   * @param stream CUDA stream used for device memory operations and kernel launches
    * @return The chars column
    */
-  [[nodiscard]] column_view chars() const;
+  [[deprecated]] [[nodiscard]] column_view chars(
+    rmm::cuda_stream_view stream = cudf::get_default_stream()) const;
```

Comment:
If it is deprecated, why its signature is changed now? And I also see that its implementation is deleted from the source file.

Reply:
Removing this completely will be done in the next release. 
The signature is not _really_ changed since the parameter is defaulted and so should cause no upstream breakages.